[2025-02-21T14:27:04.594+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-21T14:26:59.993378+00:00 [queued]>
[2025-02-21T14:27:04.630+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-21T14:26:59.993378+00:00 [queued]>
[2025-02-21T14:27:04.631+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-02-21T14:27:04.688+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): kafka_data_stream> on 2025-02-21 14:26:59.993378+00:00
[2025-02-21T14:27:04.709+0000] {standard_task_runner.py:57} INFO - Started process 963 to run task
[2025-02-21T14:27:04.736+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'kafka_spark_dag', 'kafka_data_stream', 'manual__2025-02-21T14:26:59.993378+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/dag_kafka_spark.py', '--cfg-path', '/tmp/tmpmtjl8tuj']
[2025-02-21T14:27:04.749+0000] {standard_task_runner.py:85} INFO - Job 40: Subtask kafka_data_stream
[2025-02-21T14:27:04.967+0000] {task_command.py:416} INFO - Running <TaskInstance: kafka_spark_dag.kafka_data_stream manual__2025-02-21T14:26:59.993378+00:00 [running]> on host 9240cfed0829
[2025-02-21T14:27:05.280+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='kafka_spark_dag' AIRFLOW_CTX_TASK_ID='kafka_data_stream' AIRFLOW_CTX_EXECUTION_DATE='2025-02-21T14:26:59.993378+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-21T14:26:59.993378+00:00'
[2025-02-21T14:27:52.546+0000] {logging_mixin.py:154} INFO - Dataset downloaded successfully.
[2025-02-21T14:27:52.863+0000] {logging_mixin.py:154} INFO - Dataset extracted successfully.
[2025-02-21T14:27:52.864+0000] {logging_mixin.py:154} INFO - No CSV file found in extracted data.
[2025-02-21T14:27:52.867+0000] {logging_mixin.py:154} INFO - Dataset preparation failed.
[2025-02-21T14:27:52.875+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-21T14:27:52.912+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=kafka_spark_dag, task_id=kafka_data_stream, execution_date=20250221T142659, start_date=20250221T142704, end_date=20250221T142752
[2025-02-21T14:27:53.004+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-21T14:27:53.094+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
